### ISSUE TITLE
End-to-end tests require unresolvable dependencies

### ISSUE THREAD

**Issue by holiday-sunrise** [mulesoft/mule-migration-assistant#482]
fix: add exclusions for unsupported dependencies
```
            <dependency>
                <groupId>com.mulesoft.mule.distributions</groupId>
                <artifactId>mule-system-tests</artifactId>
                <version>${mule.runtime.version}</version>
                <type>test-jar</type>
                <scope>test</scope>
                <exclusions>
                    <exclusion>
                        <groupId>com.ibm.jdbc</groupId>
                        <artifactId>db2jcc</artifactId>
                    </exclusion>
                    <exclusion>
                        <groupId>com.microsoft.jdbc</groupId>
                        <artifactId>sqljdbc4</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>
```

**Comment by afelisatti** [comment]
Hi, @holiday-sunrise. Thanks for reporting this and your recent contributions! We've analyzed the issue here, and we believe we have a bigger problem to resolve around the end-to-end tests currently handled by a specific profile. Like you mention, they require unresolvable dependencies. Did you stumble on this trying to use the profile?
Unfortunately, fixing this will involve more work than the exclusions, so we will close the matching PR while reviewing our best course of action.

**Comment by svacas** [comment]
the new end to end test strategy removed all unresolvable dependencies
fixed by 2bc200f85997bbe0f994b1912e3df7a04b8a9600

**Comment by holiday-sunrise** [comment]
In which issue did you solve it ?
